### DISCUSSION TITLE
json in websocket

### DISCUSSION THREAD

**Discussion by Menzer05** [actix/actix-web#1910] [Q&A] [answered]
please, how to send json data in wsbsocket

**Comment by robjtede** [comment, marked as answer]
using serde_json encode to json and send as text

**Reply by Menzer05** [reply]
thank s Robjted

**Reply by Menzer05** [reply]
thanks, but since my json comes from a database (influxdb) the compiler tells me it's a future and refuses to send it as text, please help me
